Makefile

```diff
@@ -399,14 +398,17 @@ dist-clean: clean
 	@go clean -testcache
 endif
 
-
 PHONY += functional-check
 functional-check: check-for-root-user build
 	@for test in $$(find $$FUNCTIONAL_TEST_DIR -name "*.bats" | sort); do \
 		echo "Running $$test"; \
 		bats $$test; \
 	done
 
+PHONY += dev-check-all
+dev-check-all: check-for-root-user build lint functional-check check
+	@echo "Running dev tests have passed";
```

Comment:
You don't need the trailing semi-colon as there is only one command present.

Reply:
Thanks..Done!